e2e/browser/helpers/authorizeClientApp.ts

```diff
@@ -0,0 +1,20 @@
+import { Selector, t } from "testcafe";
+import FetchPage from "../page-models/FetchPage";
+
+export async function authorizeNss() {
+  // Authorize our client application to access Pod resources. On NSS this will
+  // only be required the first time this app is seen at the Pod. So we need a
+  // conditional check here.
+  const authorizeButtonExists = await Selector("[name=consent]").exists;
+  if (authorizeButtonExists) {
+    // Click here to grant Control access to our app. Only certain apps may need
+    // that level of access, so don't provide by default.
+    // await t.click("#control");
```

Comment:
Not yet, at least the TestCafe tests currently don't require Control access (yet).  But that's why I left this comment in here - once these tests are extended to do lots more stuff, this will certainly be needed. I'm just wary of extending the set of tests we do here too much, 'cos the Solid Spec Test Suite needs to be doing basically lots of the same tests, so I don't want to replicate too much work.